I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/dialogues/corridor.rpy", line 12, in script
    $ renpy.jump("corridor_" + str(position))
ScriptError: could not find label 'corridor_0'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "D:\downloads\renpy-8.2.1-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "D:\downloads\renpy-8.2.1-sdk\renpy\main.py", line 635, in main
    run(restart)
  File "D:\downloads\renpy-8.2.1-sdk\renpy\main.py", line 145, in run
    renpy.execution.run_context(True)
  File "D:\downloads\renpy-8.2.1-sdk\renpy\execution.py", line 955, in run_context
    context.run()
  File "game/dialogues/corridor.rpy", line 12, in script
    $ renpy.jump("corridor_" + str(position))
  File "D:\downloads\renpy-8.2.1-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'corridor_0'.

Windows-10-10.0.22631 AMD64
Ren'Py 8.2.1.24030407
Staszic The Game 1.0
Sun Mar 17 22:24:47 2024
